WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -136,11 +136,11 @@ class PostListViewModel @Inject constructor(
     private val uploadStatusMap = HashMap<Int, PostListItemUploadStatus>()
     private val featuredImageMap = HashMap<Long, String>()
 
-    // Keep a reference to the currently being trashed post, so we can hide it during Undo SnackBar
-    private var postIdToTrash: Pair<LocalPostId, RemotePostId>? = null
+    // Keep a reference to the post currently being trashed/restored, so we can change it's ui state
```

Comment:
NitPick: we should update the comment to plural

Reply:
Comment removed as part of 244e84938e455a0f8033b7fae3413a0dd21d3762.